src/components/webView.re

```diff
@@ -60,10 +67,63 @@ let make =
       ~style=?,
       ~renderError=?,
       ~renderLoading=?,
-      ~onError=?,
-      ~onLoad=?,
-      ~onLoadEnd=?,
-      ~onLoadStart=?,
+      ~onError:
+         option(
+           {
+             .
+             "url": string,
+             "title": string,
+             "loading": bool,
+             "canGoBack": bool,
+             "canGoForward": bool,
+             /* iOS only */
+             "domain": Js.Null_undefined.t(string),
```

Comment:
`Null_undefined` is AFAIK deprecated. We should probably use `Js.Nullable` instead